cs/test/core/JsonParsingTests.cs

```diff
@@ -703,5 +703,25 @@ static T ParseJson<T>(string json) where T : new()
             var target = deserializer.Deserialize<T>(reader);
             return target;
         }
+
+        [Test]
+        public void JsonParsing_DateAsStringInStreamConstructor()
+        {
+            string timestamp = "2017-03-07T04:47:50.1145227Z";
+            string json = $"{{\"timestamp\":\"{timestamp}\"}}";
+
+            DateAsString target;
+            using (MemoryStream ms = new MemoryStream())
```

Comment:
The Travis CI build failed for as the `reader` constructed here never gets used. My proposed change will fix this problem as well.